### PR TITLE
Setting argspecs for the edpm_sshd role

### PR DESCRIPTION
**Pull request by jpodivin**
This also provides documentation for the role.

### CODE REVIEW

**Comment by jpodivin** [comment]
recheck

**Comment by jpodivin** [comment]
recheck

**Review by rebtoor — COMMENTED** [review]
+2

**Comment by jpodivin** [comment]
recheck

**Comment by jpodivin** [comment]
recheck

**Comment by jpodivin** [comment]
recheck

**Comment by jpodivin** [comment]
recheck

File: roles/edpm_sshd/meta/argument_specs.yml [hunk 1]
```diff
@@ -11,5 +11,60 @@ argument_specs:
...
+      edpm_sshd_banner_text:
         type: str
-        required: true
+        default: ''
+      edpm_sshd_password_authentication:
+        type: bool
+        default: False
+        description: Enable password authentication?
+      edpm_sshd_server_options:
+
```
Comment (rebtoor) [inline roles/edpm_sshd/meta/argument_specs.yml:32]: is this white line needed?

**Review by rebtoor — COMMENTED** [review]
just a "cosmetic" comment, otherwise +2

File: roles/edpm_sshd/meta/argument_specs.yml (hunk 1 above)
Comment (jpodivin) [reply roles/edpm_sshd/meta/argument_specs.yml:32]: Nope. Nice catch.

**Review by rebtoor — APPROVED** [review]

**Comment by rebtoor** [comment]
recheck

**Comment by jpodivin** [comment]
recheck

**Comment by rebtoor** [comment]
recheck

**Pull request merged**